main.qml

```diff
@@ -372,11 +372,12 @@ ApplicationWindow {
             // here we show confirmation popup;
 
             transactionConfirmationPopup.title = qsTr("Confirmation") + translationManager.emptyString
-            transactionConfirmationPopup.text  = qsTr("Please confirm transaction:\n\n")
+            transactionConfirmationPopup.text  = qsTr("Please confirm transaction:\n")
                         + qsTr("\nAddress: ") + address
                         + qsTr("\nPayment ID: ") + paymentId
-                        + qsTr("\nAmount: ") + walletManager.displayAmount(transaction.amount)
+                        + qsTr("\n\nAmount: ") + walletManager.displayAmount(transaction.amount)
```

Comment:
That adds a line between Payment ID and Amount, which makes it, in my opinion, more clear. Thus, with this change you have:

Address:
Payment ID:

Amount:
Fee:

Mixin: